exchange/events.go

```diff
@@ -4,7 +4,7 @@ import (
 	"encoding/json"
 	"time"
 
-	jsonpatch "github.com/evanphx/json-patch"
+	"github.com/evanphx/json-patch"
```

Comment:
I didn't expect this change. Was this automatically done by GoLand?

Reply:
Probably go fmt or IntellijIdea autofix. It works without it. Should I revert it back?